libpod/container_api.go

```diff
@@ -184,6 +184,12 @@ func (c *Container) StopWithTimeout(timeout uint) error {
 		return define.ErrCtrStopped
 	}
 
+	if c.ensureState(define.ContainerStateConfigured) {
```

Comment:
I think this also has to match ContainerStateExited

Reply:
@Luap99 that is already handled here https://github.com/containers/podman/blob/main/libpod/container_api.go#L184 and `ErrCtrStopped` is silently handled on frontend.

I just wanted to return nil for `define.ContainerStateConfigured` since container never existed in OCI runtime to it cant be stopped although same behaviour can also be achived by returning `ErrCtrStopped` here. Let me know if `ErrCtrStopped` sounds more convincing for `define.ContainerStateConfigured` as well. I am cool with both of them.

Reply:
@Luap99 this is already handled marking this as resolved.